lib/kredis/types/list.rb

```diff
@@ -3,12 +3,12 @@
 class Kredis::Types::List < Kredis::Types::Proxying
   prepend Kredis::DefaultValues
 
-  proxying :lrange, :lrem, :lpush, :ltrim, :rpush, :exists?, :del
+  proxying :lrange, :lrem, :lpush, :ltrim, :rpush, :exists?, :del, :llen
 
   attr_accessor :typed
 
-  def elements
-    strings_to_types(lrange(0, -1) || [], typed)
+  def elements(start = 0, stop = -1)
```

Comment:
Rather than give slicing arguments to `#elements`, would introduce `#slice` and have `#elements` call it.